app/views/shutmucr/shut_mucr_summary.scala.html

```diff
@@ -44,16 +44,7 @@
                         ),
                         value = Value(
                             content = Text(mucr)
-                        ),
```

Comment:
Shouldn't this change be based on a feature flag?
By removing here, we will lose the ability to change it for all journeys (with feature flag enabled and disabled)

Reply:
It was agreed with Ian to remove the change button from all journeys on the internal frontend. Arrive, Depart, etc. have already been updated, this one was forgotten. Additionally, if the user needs to change the consignment, the back button lets her/him find a new consignment from the query (initial) page.

Reply:
Cool! :) Thank you for the info